server/src/main/java/org/elasticsearch/cluster/metadata/MappingMetadata.java

```diff
@@ -33,7 +32,10 @@
  */
 public class MappingMetadata extends AbstractDiffable<MappingMetadata> {
 
-    public static final MappingMetadata EMPTY_MAPPINGS = new MappingMetadata("_doc", Collections.emptyMap());
+    public static final MappingMetadata EMPTY_MAPPINGS = new MappingMetadata(
+        MapperService.SINGLE_MAPPING_NAME,
+        Map.of(MapperService.SINGLE_MAPPING_NAME, Map.of())
```

Comment:
I adjusted this to have consistency with empty mappings that were manually/externally (and would be wrapped with `_doc`) created when using this constant.